scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -876,19 +876,20 @@ class FormatOps(
   )(implicit style: ScalafmtConfig): Boolean = {
     def opensImplicit =
       (style.newlines.forceAfterImplicitParamListModifier ||
-        next(ft).hasBreak) && opensConfigStyleImplicitParamList(ft)
+        next(ft).hasBreak) && opensConfigStyleImplicitOrUsingParamList(ft)
     (ft.hasBreak || opensImplicit) && {
       val close = matching(ft.left)
       tokens(close, -1).hasBreak
     }
   }
 
-  def opensConfigStyleImplicitParamList(
+  def opensConfigStyleImplicitOrUsingParamList(
```

Comment:
perhaps i'd suggest not renaming these methods but in scaladoc adding a comment to specify it covers "using" as well. same for other cases.